docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -207,7 +204,6 @@ import jakarta.ws.rs.core.SecurityContext;
 import org.eclipse.microprofile.jwt.JsonWebToken;
 
 @Path("/secured")
-@RequestScoped
```

Comment:
It will work with `@RequestScoped`, but what do you mean that this is not relevant at this stage of the guide ?

Reply:
@nkolatsis Oh, sorry, there are a few `TokenSecuredResource` variations and the one you are updating would not be impacted. 
However, I'd prefer to use this context annotation here, as other resources below still have it, and there is a dedicated `Supported injection scopes` section below which talks about it.

May be it is worth expanding that section instead, currently it gives only a single example, and says you don't have to use `RequestScoped` for `JsonWebToken` - but the example is missing. So if you'd like add an example there showing how the same `birthday` claim can be accessed from `JsonWebToken` without `@RequestScoped`

Reply:
I only looked at the actual quickstart code where removing it would be a problem, so got confused, sorry about that